### PR TITLE
Document key-auth TTL parameter

### PR DESCRIPTION
**Pull request by B-Galati**
### Summary
Document `ttl` parameter when creating a key with the key-auth plugin.

### Reason
This feature is not documented

### CODE REVIEW

File: app/_hub/kong-inc/key-auth/index.md
Comment (drochette) [inline app/_hub/kong-inc/key-auth/index.md:229]:
```suggestion
`ttl`<br>*optional*               | The number of seconds the key is going to be valid. If missing the ttl is unlimited.
```

Comment (tlblessing) [reply app/_hub/kong-inc/key-auth/index.md:229]: Thank you for the PR. I edited the suggestion and have requested review from core team for technical accuracy. cc @kikito @gszr

Comment (tlblessing) [reply app/_hub/kong-inc/key-auth/index.md:229]: Direct review link:  https://deploy-preview-2873--kongdocs.netlify.app/hub/kong-inc/key-auth/#create-a-key

Also, the edit does not seem to appear in the build at this time. Perhaps the formatting is off.

image

This PR is an edit to the documentation. The description says the feature is not documented, so this is an improvement to the docs.

Comment (tlblessing) [reply app/_hub/kong-inc/key-auth/index.md:229]: I reran the build but still the edit does not appear.

**Comment by tlblessing** [comment]
@Kong/team-core can someone please confirm the technical accuracy of this doc contribution?

**Comment by tlblessing** [comment]
https://deploy-preview-2873--kongdocs.netlify.app/hub/kong-inc/key-auth/#create-a-key

image

**Comment by B-Galati** [comment]
I just rebased the branch, squashed the commits and improve de the wording, let me know for anything.

**Comment by B-Galati** [comment]
And I also edited Key-Auth Enc plugin

**Review by Bradamant3 — APPROVED** [review]
LGTM and TYSM @B-Galati, with apologies for the delay in approving.

**Pull request merged**
